ait/dsn/bin/examples/cltu_api_test.py

```diff
@@ -38,9 +38,9 @@
 import ait.dsn.sle
 
 cltu_mngr = ait.dsn.sle.CLTU(
-    hostname='atb-ocio-sspsim.jpl.nasa.gov',
+    hostname='127.0.0.1', #test for now
     port=5100,
-    inst_id='sagr=LSE-SSC.spack=Test.fsl-fg=1.cltu=cltu1',
+    inst_id='sagr=39.spack=default.fsl-fg=1.cltu=cltu1',
```

Comment:
Same comment as above.

Reply:
the original "LSE-SSC" is also project specific, but from another project. Should I add another field in config.yaml for CLTU_inst_id?